services/grafana-loki/metadata.yaml

```diff
@@ -2,6 +2,8 @@ displayName: Grafana Loki
 description: Horizontally-scalable, highly-available, multi-tenant log aggregation system inspired by Prometheus. Requires Logging Operator.
 category:
   - logging
+dependencies:
```

Comment:
To collect and query logs out of the box, users need to install these:

* `logging-operator` - configures fluentd/fluentbit to push logs to Loki
* `grafana-logging` - grafana UI for querying Loki
*  `rook-ceph`/`rook-ceph-cluster` - for backend storage, but you can configure Loki to use another backend like s3